src/app/tests/suites/certification/Test_TC_MF_1_29.yaml

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    22.1.29. [TC-MF-1.29] Open commissioning window - normal operation and
+    closing [BCM] [DUT - Commissionee]
+
+config:
+    nodeId: 0x12344321
+    timeout: 330
+    endpoint: 0
+
+tests:
+    - label: "Factory Reset the DUT_CE"
+      cluster: "SystemCommands"
+      command: "FactoryReset"
+
+    - label: "TH_CR1 starts a commissioning process with DUT_CE"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH_CR1 opens a commissioning window on DUT_CE"
+      cluster: "AdministratorCommissioning"
+      command: "OpenBasicCommissioningWindow"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "CommissioningTimeout"
+                value: 300
+
+    - label: "Wait for commissioning Window to 301 seconds"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 50000
+
+    - label: "Wait for commissioning Window to 301 seconds"
```

Comment:
You don't need this repeated-delay thing anymore now that https://github.com/project-chip/connectedhomeip/pull/17906 is fixed.

Reply:
i will update it